storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/RxQueryTest.java

```diff
@@ -168,4 +172,19 @@ public void queryShouldBeUpdatedAfterDelete() {
         emissionChecker.assertThatNoExpectedValuesLeft();
         subscription.unsubscribe();
     }
+
+    @Test
+    public void shouldThrowExceptionIfCursorNullObservable() {
+        final PreparedGetCursor queryWithNullResult = createQueryWithNullResult();
+
+        final TestSubscriber<Cursor> testSubscriber = new TestSubscriber<Cursor>();
+
+        queryWithNullResult
+                .createObservable()
+                .subscribe(testSubscriber);
+
+        testSubscriber.awaitTerminalEvent();
+        testSubscriber.assertNoValues();
+        testSubscriber.assertError(StorIOException.class);
```

Comment:
Please add check for exception cause type and message, we must be sure that it's really expected exception.